### ISSUE TITLE
what is the accuracy for wav2vec 2.0 pretrained model?

### ISSUE THREAD

**Issue by LamyaMohamed** [pytorch/fairseq#2949]
What is the accuracy for identifying the correct latent representation both training and testing?
in the paper it is mentioned that it is 62% (?)

> The training accuracy of identifying the correct latent audio representation increases from 62% to 78.0% when switching from quantized to continuous targets.

But is it for the base or large model? and is it for 960 hrs or 53k hrs of training? and 62% is considered very low..

Thank you..

**Comment by alexeib** [comment]
accuracy during pretraining and performance on downstream task are not necessarily correlated - especially across model architectures. you could, for example, change mask length to 1 and get close to 100% accuracy, but your downstream task performance will be very bad.

if i remember correctly, the base model on librispeech gets around 62%, while the large model on librivox gets around 70%

**Comment by rajeevbaalwan** [comment]
> accuracy during pretraining and performance on downstream task are not necessarily correlated - especially across different configurations (and model architectures). you could, for example, change mask length to 1 and get close to 100% accuracy, but your downstream task performance will be very bad.
> 
> if i remember correctly, the base model on librispeech gets around 62%, while the large model on librivox gets around 70%

@alexeib this 62% accuracy is for wav2vec or wav2vec2.0 Base model on LS-960 ?? In the paper you have mentioned the following:
**"The training accuracy of identifying the correct latent audio representation increases from 62% to 78.0% when switching from quantized to continuous targets."**

Looks like in above mentioned line in the paper you are comparing the LS-960 pretraining accuracy of the wav2vec-Base and wav2vec2.0-Base Models.

Also this 62% is on validation set or training set ?
